bin/behat-prepare.sh

```diff
@@ -169,78 +145,59 @@ foreach (\$config['example-userpass'] as \$key => &\$user) {
-cp "$BASH_DIR"/fixtures/shib13-sp-remote.php  "$PREPARE_DIR"/private/simplesamlphp/metadata/shib13-sp-remote.php
+cp "$BASH_DIR/fixtures/config.php"         "$PREPARE_DIR/private/simplesamlphp/config/config.php"
+cp "$BASH_DIR/fixtures/config-prepare.php" "$PREPARE_DIR/wp-content/mu-plugins/config-prepare.php"
+cp "$BASH_DIR/fixtures/saml20-sp-remote.php"  "$PREPARE_DIR/private/simplesamlphp/metadata/saml20-sp-remote.php"
+cp "$BASH_DIR/fixtures/saml20-idp-hosted.php" "$PREPARE_DIR/private/simplesamlphp/metadata/saml20-idp-hosted.php"
+cp "$BASH_DIR/fixtures/shib13-idp-hosted.php" "$PREPARE_DIR/private/simplesamlphp/metadata/shib13-idp-hosted.php"
+cp "$BASH_DIR/fixtures/shib13-sp-remote.php"  "$PREPARE_DIR/private/simplesamlphp/metadata/shib13-sp-remote.php"
 
-# Enable the exampleauth module
-touch "$PREPARE_DIR"/private/simplesamlphp/modules/exampleauth/enable
+touch "$PREPARE_DIR/private/simplesamlphp/modules/exampleauth/enable"
 
-# Generate a certificate SimpleSAMLphp uses for encryption
-# Because these files are in ~/code/private, they're inaccessible from the web
-openssl req -newkey rsa:2048 -new -x509 -days 3652 -nodes -out "$PREPARE_DIR"/private/simplesamlphp/cert/saml.crt -keyout "$PREPARE_DIR"/private/simplesamlphp/cert/saml.pem -batch
+openssl req -newkey rsa:2048 -new -x509 -days 3652 -nodes \
+  -out "$PREPARE_DIR/private/simplesamlphp/cert/saml.crt" \
+  -keyout "$PREPARE_DIR/private/simplesamlphp/cert/saml.pem" -batch
 
 TWIG_TEMPLATE_PATH="$PREPARE_DIR/private/simplesamlphp/modules/core/templates/loginuserpass.twig"
-# Modify the login template so Behat can submit the form
 echo "Operating on: $TWIG_TEMPLATE_PATH"
-sed -i  -- "s/<button class=\"pure-button pure-button-red pure-input-1-2 pure-input-sm-1-1 right\" id=\"submit_button\"/<button class=\"pure-button pure-button-red pure-input-1-2 pure-input-sm-1-1 right\" id=\"submit\"/g" "$TWIG_TEMPLATE_PATH"
-sed -i  -- "s/Login/Submit/g" "$TWIG_TEMPLATE_PATH"
-# Modify the loginuserpass.js file so Behat can submit the form
-JS_FILE_PATH="$PREPARE_DIR/private/simplesamlphp/modules/core/public/assets/js/loginuserpass.js" # Adjust path if necessary
-sed -i -- 's/getElementById("submit_button")/getElementById("submit")/g' "$JS_FILE_PATH"
-sed -i -- 's/button.disabled = true;//g' "$JS_FILE_PATH"
+sed -i -- 's/id="submit_button"/id="submit"/g' "$TWIG_TEMPLATE_PATH" || true
+sed -i -- 's/>Login</>Submit</g' "$TWIG_TEMPLATE_PATH" || true
 
-composer install --no-dev --working-dir="$PREPARE_DIR"/private/simplesamlphp --ignore-platform-req=ext-ldap
+JS_FILE_PATH="$PREPARE_DIR/private/simplesamlphp/modules/core/public/assets/js/loginuserpass.js"
+sed -i -- 's/getElementById("submit_button")/getElementById("submit")/g' "$JS_FILE_PATH" || true
+sed -i -- 's/button.disabled = true;//g' "$JS_FILE_PATH" || true
+
+composer install --no-dev --working-dir="$PREPARE_DIR/private/simplesamlphp" --ignore-platform-req=ext-ldap || true
```

Comment:
Why was this changed? These particular lines of code went through several iterations to ensure that they matched correctly to the simplesamlphp login page templates.